tests/acceptance/api_test.go

```diff
@@ -700,7 +700,7 @@ func TestApi(t *testing.T) {
 					// Verify error response json
 					// Get nested json in response json
 					errResp := data["error"].(map[string]interface{})
-					assert.Equal(r, "ClientError", errResp["code"].(string))
+					assert.Equal(r, "RequestValidationError", errResp["code"].(string))
```

Comment:
What would you think about making this a `404 Not Found` error instead?
The client is requesting to delete a specific resource, and that resource doesn't exist -- I would usually consider that a 404.

Reply:
Fixed